### ISSUE TITLE
Module doesn't exist

### ISSUE THREAD

**Issue by therealOri** [backbone-hq/pqcrypto#10]
I Pip installed the package, imported like what's showing for the examples, tried to run and I get the following error.

```bash
Traceback (most recent call last):
  File "/home/ori/Desktop/playground/latty/latty.py", line 1, in <module>
    from pqcrypto.kem.kyber1024_90s import generate_keypair, encrypt, decrypt
  File "/home/ori/Desktop/playground/latty/pqlENV/lib/python3.10/site-packages/pqcrypto/kem/kyber1024_90s.py", line 1, in <module>
    from .._kem.kyber1024_90s import ffi as __ffi, lib as __lib
ModuleNotFoundError: No module named 'pqcrypto._kem.kyber1024_90s'
```

Anything in the `_kem` directory is not there. It's empty. (Except for the very empty .gitkeep file)

**Comment by ttinies** [comment]
TLDR: execute compile.py in the source code before pip installing pqcrypto. The instructions as is are incomplete.

I don't really have anything to do with this repository, but I found a __main__ statement in compile.py of this source code that would build/compile the project, but that isn't seem to be invoked in the poetry build process nor specified in procedure outlined on the project's main page.  It might look something like the following given the build code's current state:

> # Using poetry
> pip install poetry
> _**python compile.py**_
> poetry build

**Comment by Erebus9456** [comment]
Even after following your steps it doesn't work. Same error still. Can you help me please?

**Comment by longh30kma** [comment]
I tried and succeeded thanks

**Comment by neomn** [comment]
ok , this is how to fix this error after installing using pip 
1- download the source code 
2- cd into the source code directory 
3- sudo python3 compile.py (may take a while to compile) , this will create a _kem directory in ./pqcrypto/
4- go to the package directory downloaded by pip , usually in /home/$USER/.local/lib/python3.11/site-packages/pqcrypto
5- create a __init__.py file in kem directory 
6- remove _kem directory 
7- cp _kem from compiled source code into the downloaded package by pip

**Comment by neomn** [comment]
> ok , this is how to fix this error after installing using pip 1- download the source code 2- cd into the source code directory 3- sudo python3 compile.py (may take a while to compile) , this will create a _kem directory in ./pqcrypto/ 4- go to the package directory downloaded by pip , usually in /home/$USER/.local/lib/python3.11/site-packages/pqcrypto 5- create a **init**.py file in kem directory 6- remove _kem directory 7- cp _kem from compiled source code into the downloaded package by pip

@Erebus9456

**Comment by neomn** [comment]
@kpdemetriou 
that would be great, thanks

**Comment by Duongguyen** [comment]
> Mình đã thử và thành công thanks

help me

**Comment by neomn** [comment]
@Duongguyen 
[Checkout my package](https://github.com/neomn/NeoCrypto) , is very simple and easy to use
